option optimize_for = LITE_RUNTIME;

message FenjieAwakenItemRequest
{
 required uint32 pos = 1;
 required uint32 num = 2;
}
